ForageBuddy/Form1.cs

```diff
@@ -48,41 +99,53 @@ private void btnCalculate_Click(object sender, EventArgs e)
 
             var tmpPlayerScores = new List<PlayerScore>();
 
+            _logger.Information("New images found:");
             foreach (var file in allFilesInOutputDir.Where(x => !_filesInScreenshotDirectory.Contains(x)))
             {
-                tmpPlayerScores.AddRange(_imageParser.GetPlayerScoresInImage(file));
+                _logger.Information($"  {file.ToString()}");
+                tmpPlayerScores.AddRange(_imageParser.GetPlayerScoresInImage(file, _logger));
             }
 
             foreach (var score in tmpPlayerScores)
             {
-                if (!_playerScores.ContainsKey(score.PlayerName)) _playerScores.Add(score.PlayerName, score);
-                
+                if (!_playerScores.ContainsKey(score.PlayerName))
+                {
+                    _playerScores.Add(score.PlayerName, score);
+                    _logger.Information(score.PlayerScoreStringBreakdown());
+                }
+
                 if (score.TotalScore > _playerScores[score.PlayerName].TotalScore)
                 {
                     _playerScores[score.PlayerName] = score;
```

Comment:
This needs some logging - IE, found greater score. Adding : Score.PlayerScoreStringBreakdown()